Multisig/Logic/Ethereum/ENS.swift

```diff
@@ -63,7 +63,7 @@ final class ENS {
 
     func name(for address: Address) throws -> String? {
         // construct a reverse node
-        let addressString = address.hexadecimal
+        let addressString = String(address.hexadecimal.dropFirst(2)) // skip 0x
```

Comment:
I think, it is better to work with data directly than manipulate strings here. I'd do the following:

```swift
extension Address {
  var data: Data {
      Data(rawAddress)
  }
}

// and on line 66 here:
let addressString = address.data.toHexString()
```

Reply:
ok